app/src/main/kotlin/com/wire/android/ui/authentication/login/LoginScreen.kt

```diff
@@ -94,7 +101,21 @@ private fun LoginContent(
                 WireTabRow(
                     tabs = LoginTabItem.values().toList(),
                     selectedTabIndex = pagerState.calculateCurrentTab(),
-                    onTabChange = { scope.launch { pagerState.animateScrollToPage(it) } },
+                    onTabChange = {
+
+                        if (viewModel.serverConfig.apiProxy != null) {
```

Comment:
how about moving this flag to a viewmodel and check for a null there and expose it through a state or something ?